src/input-group.scss

```diff
@@ -60,7 +60,10 @@ $fd-input-border-radius: var(--sapField_BorderCornerRadius);
     @include fd-reset-child-spacing();
 
     border: none;
-    flex: 1;
+    flex: none;
```

Comment:
`flex: none` is a shorthand for: `flex-grow: 0; flex-shrink: 0; flex-basis: auto;` and then grow and shrink values are additionally overwritten. 
I'd suggest using: `flex: 1 1 auto`